contracts/finality/src/contract.rs

```diff
@@ -7,23 +7,27 @@ use crate::queries::query_block_voters;
 use crate::state::config::{get_config, Config, ADMIN, CONFIG, IS_ENABLED};
 use crate::state::public_randomness::{get_first_pub_rand_commit, get_last_pub_rand_commit};
 use babylon_bindings::BabylonQuery;
-use cosmwasm_std::{
-    to_json_binary, Deps, DepsMut, Env, MessageInfo, QueryResponse, Response, StdResult,
-};
+use cosmwasm_std::{to_json_binary, Deps, DepsMut, Env, MessageInfo, QueryResponse, Response};
 use cw_controllers::AdminError;
 
 pub fn instantiate(
     mut deps: DepsMut<BabylonQuery>,
     _env: Env,
     _info: MessageInfo,
     msg: InstantiateMsg,
-) -> StdResult<Response<BabylonMsg>> {
+) -> Result<Response<BabylonMsg>, ContractError> {
```

Comment:
L18 does not need to be changed imo

Reply:
Hmm checked other endpoint functions and all of them return `Result<Response<BabylonMsg>, ContractError>`. So all good, sorry for the confusion